Sample_v3/LoginViewController.swift

```diff
@@ -136,15 +136,11 @@ extension LoginViewController {
                 $0.channelNameLabel.backgroundColor = .yellow
             }
             
-            let navigation = UINavigationController(rootViewController: channelList)
+            // An example of using the custom router
+            channelList.router = MyChatChannelListRouter(rootViewController: channelList)
```

Comment:
I think this should be configured via `UIConfig`

Reply:
router is taken from config in ecd8af0b4ca1b2819df422cb35162cb072eca5ea